.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "daily"
+    time: "11:00"
+  open-pull-requests-limit: 10
+  assignees:
+    - "dependabot"
+
+- package-ecosystem: npm
+  directory: "/"
```

Comment:
```suggestion
  directory: "/"
  default_assignees:
    - "jawnsy"
```

Reply:
I'd be happy to be the default assignee, but I'm not sure that's actually what you want, because I don't know this project as well as you do, and so wouldn't know whether upgrades are likely to break things. I would need some guidance from you regarding what is needed when deciding whether to approve/merge - what sort of manual testing do I need to do? Or can I just take a quick look at the changes and approve/merge provided that tests are passing?

Reply:
> Or can I just take a quick look at the changes and approve/merge provided that tests are passing?

If CI is passing, then it should be okay for you to merge. If it breaks something than I will take responsibility for fixing it and then it will for us to add more tests in order to continue being able to rely on CI and our testing suites.

Reply:
lmao, living dangerously, I like it :joy:

Reply:
The only way to live 😉

Reply:
Okay feel free to make the changes and then we can approve and merge!

Reply:
Applying this change, but I think you're using the old dependabot syntax (Dependabot Preview), and not the new one, since they were acquired/integrated into GitHub: https://docs.github.com/en/github/administering-a-repository/customizing-dependency-updates

I think we'd want to set me as reviewer, then?

Reply:
Oh huh I just pulled that from their docs actually so maybe it's outdated. https://dependabot.com/docs/config-file/#default_assignees

You're right though, we want you as the reviewer (and the assignee, probably). Good catch!

Reply:
Yeah, that's for "Dependabot Preview", then they were acquired by GitHub and it became "Dependabot Native" or whatever... at least that's what I was able to gather based on the docs lol

Reply:
Ah, that's confusing. Totally missed the yellow box saying that at the top lol